src/firebase/firebase-client.service.ts

```diff
@@ -78,7 +78,7 @@ export class FirebaseClientService implements OnModuleInit {
   };
 
   mapUserData(userCredential: UserCredential) {
```

Comment:
move the map functionality from firebase service to user service and use in auth service